locale/pt-br/texts.po

```diff
@@ -21,25 +30,337 @@ msgstr ""
 "de tokens aqui."
 
-#: src/screens/InitWallet.js:66
+#: src/screens/About.js:65
+msgid "For further information, check our website"
+msgstr "Para mais informações, visite nosso site "
+
+#: src/screens/BackupWords.js:224
+msgid "To make sure you saved,"
+msgstr "Para conferir que você salvou,"
+
+#: src/screens/BackupWords.js:226
+msgid "Please select the word that corresponds to the number below:"
+msgstr "Por favor, selecione a palavra que corrende ao número abaixo:"
+
+#: src/screens/ChangePin.js:103
+msgid "Incorrect PIN code."
+msgstr "PIN incorreto."
+
+#: src/screens/ChangePin.js:130
+msgid "PIN codes don't match."
+msgstr "Os PINs estão diferentes."
+
+#.  Should never get here because we've done all the validations before
+#: src/screens/ChangePin.js:155
+msgid "Error while changing PIN"
+msgstr "Erro ao mudar o PIN"
+
+#: src/screens/ChangePin.js:161
+msgid "Please enter your **current PIN**"
+msgstr "Por favor entre com o seu **PIN atual**"
+
+#: src/screens/ChangePin.js:174
+msgid "Please enter your **new PIN**"
+msgstr "Por favor, entre com o seu **novo PIN**"
+
+#: src/screens/ChangePin.js:187
+msgid "Please enter your **new PIN** again"
+msgstr "Por favor, entre novamente com o seu **novo PIN**"
+
+#: src/screens/ChangePin.js:202
+msgid "New PIN recorded"
+msgstr "Novo PIN salvo"
+
+#: src/screens/ChangePin.js:212
+msgid "CHANGE PIN"
+msgstr "MUDAR PIN"
+
+#: src/screens/ChoosePinScreen.js:124
+msgid "Enter your new PIN code"
+msgstr "Entre com seu novo PIN"
+
+#: src/screens/ChoosePinScreen.js:136
+msgid "Enter your new PIN code again"
+msgstr "Entrei com seu novo PIN novamente"
+
+#: src/screens/ChoosePinScreen.js:68
+msgid "Create a new PIN code,"
+msgstr "Criar um novo PIN,"
+
+#: src/screens/ChoosePinScreen.js:71
+msgid "To confirm the PIN,"
+msgstr "Para confirmar o PIN,"
+
+#: src/screens/ChoosePinScreen.js:150
+msgid "PIN codes don't match. Try again."
+msgstr "Os PINs estão diferentes. Tente novamente."
+
+#: src/screens/ChoosePinScreen.js:173
+msgid "Start the Wallet"
+msgstr "Iniciar a Wallet"
+
+#: src/screens/CreateToken.js:70
+msgid "You don't have any hathor tokens available to create your token."
+msgstr "Você não tem nenhum token Hathor disponível para criar o seu token."
+
+#: src/screens/CreateToken.js:85
+msgid "All fields must be filled."
+msgstr "Todos os campos devem ser preenchidos."
+
+#: src/screens/CreateToken.js:123 src/screens/CreateTokenConfirm.js:84
+msgid "Enter your 6-digit pin to create your token"
+msgstr "Entre com seu PIN de 6 digitos para criar o seu token"
+
+#: src/screens/CreateToken.js:124 src/screens/CreateTokenConfirm.js:85
+msgid "Authorize token creation"
+msgstr "Autorizar criação de token"
+
+#: src/screens/CreateToken.js:140
+msgid "CREATE NEW TOKEN"
+msgstr "CRIAR UM NOVO TOKEN"
+
+#: src/screens/CreateToken.js:146 src/screens/CreateTokenName.js:51
+msgid "Token Name"
+msgstr "Nome do Token"
+
+#: src/screens/CreateToken.js:153
+msgid "Token Symbol"
+msgstr ""
```

Comment:
Missing translation.

Reply:
If the intent is to have the same text, I suggest using the same text bellow. The end effect is the same, but translation tools will not pick it up as "needing translation".

Reply:
Fixed. Thanks!